src/shared/ui/pickerPrompter.ts

```diff
@@ -261,21 +279,23 @@ export class QuickPickPrompter<T> extends Prompter<T> {
     public loadItems(items: Promise<DataQuickPickItem<T>[]> | DataQuickPickItem<T>[]): Promise<void> {
         const picker = this.quickPick
 
+        if (this.hasPlaceholder) {
+            this.clearItems()
+        }
+
         if (items instanceof Promise) {
             picker.busy = true
             picker.enabled = false
+            this.pendingPromises += 1
```

Comment:
Under what circumstances will you need more than one pending promise?

Reply:
Can't think of any really. At one point I had the picker loading things in parallel to see how it'd handle, but l'll just remove for now.